project.yml

```diff
@@ -4,7 +4,7 @@ options:
 packages:
   SpruceIDWalletSdkRs:
     url: https://github.com/spruceid/wallet-sdk-rs
-    from: 0.0.4
+    from: 0.0.25
```

Comment:
Not sure if this change is needed. If this is from as in ^0.0.4 then probably not.